cmd/pbapi/main.go

```diff
@@ -66,7 +65,7 @@ func main() {
 	mr.Get("/", statusOk)
 	mr.Handle("/metrics", promhttp.Handler())
 
-	log.Info().Int("webPort", 8000).Int("metricsPort", config.Prometheus.Port).Msg("Starting new instance")
+	log.Info().Msgf("Starting new instance on port 8000 with prometheus on %d", config.Prometheus.Port)
```

Comment:
I see no value in introducing these logging fields, why would we search logs by port number?